DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -514,6 +514,18 @@
   <class name="edm::Wrapper< std::vector<edm::Ptr<pat::Jet> > >" />
   <class name="edm::Wrapper< std::vector< std::vector<edm::Ptr<pat::Jet> > > >" />
 
+  <class name="pat::PATTauDiscriminatorBase">
+    <field name="transientVector_" transient="true"/>
+  </class>
+  <class name="pat::PATTauDiscriminator"/>
```

Comment:
Ok -- if've done as advised. PATTauDiscriminator now has a class version 3. The PATTauDiscriminatorBase is a templated class.